nekoyume/Assets/_Scripts/UI/Widget/ShopBuy.cs

```diff
@@ -145,6 +140,7 @@ private void Close()
             _npc?.gameObject.SetActive(false);
             shopItems.Close();
             Find<ItemCountAndPricePopup>().Close();
+            Game.Event.OnRoomEnter.Invoke(true);
```

Comment:
Is this line intended?

Reply:
Yes. If don't invoke `OnRoomEnter`, screen will get blackout after closing this widget.

Reply:
Previously, `OnRoomEnter` was separated from `Close`, but I put it inside `Close` because I thought it was meaningless.